arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -508,6 +509,17 @@ void RocksDBEngine::collectOptions(
                          arangodb::options::Flags::Enterprise));
 #endif
 
+  options
+      ->addOption("--rocksdb.use-range-delete-in-wal",
+                  "enable range delete markers in the write-ahead log (WAL). "
+                  "potentially incompatible with older arangosync versions",
```

Comment:
should we say that it defaults to true (based on the constructor)?

Reply:
We don't need to, as the default values are shown automatically for all startup options (just try `--help` or `--help-.` to see it in effect).